orgname.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+regex="github.com/[a-z0-9A-Z]*"
+git config -l | grep -o "$regex" | while read -r line ; do
+    echo "${line:11}" | xargs echo -n > org_name.txt
```

Comment:
I think we prefer two indents by default for shell.  Maybe see what ShellCheckBear says ;-)

Reply:
Seems not, [.ci/deploy.sh](https://github.com/coala/community/blob/master/.ci/deploy.sh) use 4 as well.

Reply:
that was a horribly rushed script I mostly copied from other sources and fixed up; without code review; naughty me.

https://github.com/coala/coala-bears/blob/master/.ci/deps.sh
https://github.com/coala/coala/blob/master/.misc/deps.sh
(and others in those directories)

We dont run code linting on them yet.  I've created https://github.com/coala/coala/issues/4952 for that. (I'll publish a backlog of new tasks tomorrow)

Reply:
(this script is so small it doesnt matter; you are right we dont have a convention yet, so you dont need to update this patch to follow a non-existent coding standard)